scripts/yamale_validator.py

```diff
@@ -0,0 +1,18 @@
+# Import Yamale and make a schema object:
+import yamale
+schema = yamale.make_schema('./robust.yaml')
```

Comment:
This will prevent the script from being run outside the `scripts` directory. To allow the script to be used anywhere, you could add:
```python
import os

dir_here = os.path.dirname(__file__)
schema_path = os.path.join(dir_here, 'robust.yaml')
data_path = os.path.join(dir_here, 'yamale.bug')
```